### DISCUSSION TITLE
docking ports...

### DISCUSSION THREAD

**Discussion by FuryFire2004** [zer0Kerbal/Dreamer#37] [Q&A] [answered]
what docking ports are the ports in this mod compatible with, I might be able to write configs to allow usage with the standard clamp-o-tron port.

**Comment by FuryFire2004** [comment]
however you might need to wait like 12-15 hours given that it's like 1 am where i live right now...

**Comment by zer0Kerbal** [comment]
I haven't uploaded the current configs - more then.

**Comment by zer0Kerbal** [comment, marked as answer]
```	MODULE
	{
		name = ModuleDockingNode
		deployAnimationController = 1
		nodeType = size0
		controlTransformName = controlNode
	}
```

Am considering making it size0,size1 (jr and normal)

**Reply by FuryFire2004** [reply]
Yea

**Comment by FuryFire2004** [comment]
also there's a docking port in habtech 2 that the irl version of this would dock to which is kind of a mid size between jr and normal ports
